src/assets/MobileTestAutomation/automationFrameworksComparison.md

```diff
@@ -0,0 +1,162 @@
+|-----------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
+| **Test Execution:** Tests are stable, highly reliable, and performed faster       | **No Cross-Platform:** Limited to iOS apps                                       |
+| **Easy to Setup:** Setting up XCUITest is straightforward as works well XCode IDE | **Limited Language Support:** It supports Swift or Objective-C for writing tests |
+| **Community Support:** Built by Apple with large community                        | **Environment:** Only for macOS                                                  |
+|                                                                                   | **IDE Dependent:** XCode IDE needs to be installed                               |
+
+#### When to use?
+
+- When developing iOS applications with stable, reliable, and fast testing
+- When your team is focused on iOS development
+- When you need to test iOS apps across various devices and screen sizes
+
+## [**Calabash**](https://github.com/calabash)
+
+Calabash is an open-source framework that allows you to write and execute tests for Android and iOS apps. It uses
+Cucumber, a tool that lets you write tests in plain language.
+
+#### Pros and Cons:
+
+| **Pros 👍**                                                                                | **Cons 👎**                                                                                                    |
+|--------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|
+| **Environment:** Can be installed and run on various OS such as Windows, Linux, and macOS. | **Lack of documentation:** Can be slower compared to other tools because it uses a client-server architecture. |
+| **Cross-Platform:** It supports writing tests for both iOS and Android.                    | **Limited Language Support:** It supports only Ruby                                                            |                                                                                                                                    |
+| **User-Friendly:** Write tests in plain English, making it accessible for non-programmers. | **No Regular Updates:** Active development has ceased, which might affect future compatibility                 |
+| **Open-Source:** Free to use                                                               | **Low Community Support:** Very low community support as it is not used by many                                |
+
+#### When to use?
+
+- When your team includes non-technical members who need to write tests
+- For projects that would benefit from behavior-driven development (BDD)
```

Comment:
maybe link to one of our blogs about BDD or some other source that describes the concept of BDD. As far as I remember, there were a few sources in the other blog from Artur that you've recently updated.

Reply:
I have added a link to the BDD docs.